resources/page/pagegroup_test.go

```diff
@@ -75,15 +81,13 @@ func TestGroupByWithFieldNameArg(t *testing.T) {
 	}
 
 	groups, err := pages.GroupBy(context.Background(), "Weight")
-	if err != nil {
-		t.Fatalf("Unable to make PagesGroup array: %s", err)
-	}
-	if !reflect.DeepEqual(groups, expect) {
-		t.Errorf("PagesGroup has unexpected groups. It should be %#v, got %#v", expect, groups)
-	}
+	c.Assert(err, qt.IsNil)
+	c.Assert(groups, comparePageGroup, expect)
```

Comment:
I suspect qt.DeepEquals should do the job here?

Reply:
It doesn't, go-cmp can't compare unexported fields, so you can't compare `Page`s (I tried before making a custom comparer 😅)

Reply:
I'm just now noticing that you can use `cmpopts.IgnoreUnexported` or `AllowUnexported`, oh well. I can fix it or leave it like this, let me know.

Reply:
No, leave it, it makes sense ...